xmtp_mls/src/groups/mod.rs

```diff
@@ -297,61 +422,64 @@ where
     }
 
     pub async fn send_message(&self, message: &[u8]) -> Result<(), GroupError> {
-        let mut conn = self.client.store.conn()?;
+        let conn = &mut self.client.store.conn()?;
         let intent_data: Vec<u8> = SendMessageIntentData::new(message.to_vec()).into();
         let intent =
             NewGroupIntent::new(IntentKind::SendMessage, self.group_id.clone(), intent_data);
-        intent.store(&mut conn)?;
+        intent.store(conn)?;
 
-        self.publish_intents(&mut conn).await?;
+        // Skipping a full sync here and instead just firing and forgetting
+        self.publish_intents(conn).await?;
```

Comment:
That's right. Especially with `max_past_epochs` we should be able to assume that sending a message will "just work" in almost all cases and not need to be re-queued. There are some edge cases where your local state is way out of sync with the network where it would need a retry, but they should be rare enough I think the trade-off is worth it.